packages/interface/src/components/navigable-region/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __unstableMotion as motion } from '@wordpress/components';
+
+export default function NavigableRegion( {
+	children,
+	className,
+	ariaLabel,
+	motionProps = {},
+} ) {
+	const Tag = Object.keys( motionProps ).length ? motion.div : 'div';
+
+	return (
+		<Tag
+			className={ classnames( 'interface-navigable-region', className ) }
+			aria-label={ ariaLabel }
+			role="region"
+			tabIndex="-1"
+			{ ...motionProps }
+		>
+			<div className="interface-navigable-region__stacker">
```

Comment:
Noting that this may have caused a regression: https://github.com/WordPress/gutenberg/issues/45778

A fix could be to set the display + flex direction on this `__stacker` class, but I'm not familiar enough with the change to be sure.

Reply:
I can confirm this. The same is visible in the post editor if a theme has a non-white background.

Reply:
Looking into this!